actions/generate-schedule/main.go

```diff
@@ -112,8 +131,18 @@ func validateFlags() error {
 		}
 	}
 
-	if *userList == "" {
-		return fmt.Errorf("--users must be specified")
+	if *githubTeam != "" || *githubOrg != "" {
+		if *githubTeam == "" || *githubOrg == "" {
+			return fmt.Errorf("must specify both --githubTeam and --githubOrg")
+		}
+
+		var ok bool
+		accessToken, ok = os.LookupEnv(githubTokenEnvKey)
+		if !ok || accessToken == "" {
+			log.Fatalf("GITHUB_TOKEN environment variable cannot be empty. Get one from https://github.com/settings/tokens, and ensure it has the 'read:org' scope.")
+		}
+	} else if *userList == "" {
+		log.Fatalf("must specify either non-empty --users or (--githubTeam and --githubOrg).")
```

Comment:
Was about to comment that perhaps we could add this sentence to the validation for each flag just to make it clear the conditions under which each is required, but if we only anticipate a single client then not sure it's worth it 🤷‍♀

Reply:
Added. Also merged in the `--record` logic like I did in the Google Calendar PR